### ISSUE TITLE
Maiko cmake builds:  cmake build process does not drop required files in maiko/<osversion>.<machinetype> folder

### ISSUE THREAD

**Issue by fghalasz** [Interlisp/medley#1238]
makeright drops the lde* files in the directory maiko/<osversion>.<machinetype> , where they are expected to be by run-medley and its peers.

cmake --build leaves the lde* files in the build directory.  And cmake --install is a no-op.

Need to add a step in the cmake (either build or install) to drop the files in  maiko/<osversion>.<machinetype>.

**Comment by nbriggs** [comment]
"makeright" puts the files in `maiko/{osname}.{arch}` for the single architecture it built for.
cmake generated makefiles (or equivalent) leave them in whatever directory you ran the "cmake" and "make" in, but we have not included an install target in the maiko/CMakeLists.txt.
For now, we could have install put them in the same place that makeright does, but I don't know that that is really the right place.

**Comment by fghalasz** [comment]
My goal is to have the "output" of makeright and cmake be identical - so we can switch back and forth without having to change a lot of other infrastructure (e.g., in github workflows).

Until we decide to change the `maiko/{osname}.{arch}` scheme and update all the palces that depnd on that scheme, then I think we would be best off adding an install to cmake to place the executables into `maiko/{osname}.{arch}`.

Later we can decide if we want to update the executables location and change cmake, makeright, and everything else that depends on the executable location all at once.

**Comment by nbriggs** [comment]
Yeah.  It's looking a bit messy to get cmake to install in non-standard places.  I'm working on it.
